approval_polls/static/approval_polls/embed_instructions.js

```diff
@@ -0,0 +1,26 @@
+/* Code related to templates/approval_polls/embed_instructions.html */
+
+// Sets the iframe height and the corresponding embed instruction code in the textarea.
+function setIframeHeightAndCode(iframeId, textareaId, pollLink) {
+
+    // get the height of the contents of the iframe.
+    var iframeHeight = $(iframeId).contents().height();
```

Comment:
It's better to move all variable declarations to the top because the JavaScript interpreter will move them up anyway. See [this blog post on hoisting] (http://www.adequatelygood.com/JavaScript-Scoping-and-Hoisting.html) for a detailed explanation.